src/main/scripts/src/app/core/workspace.service.ts

```diff
@@ -30,4 +30,10 @@ export class WorkspaceService {
   public organizationCode: string;
   public projectCode: string;
 
+  public isWorkspaceSet(): boolean {
+    this.organizationCode = undefined;
```

Comment:
Sorry, this was just a test I forgot to remove. When it comes to that condition, I have tried the one you suggest but it does not work in Typescript.